joomla/joomla-cms
# [5.2] Created translatable strings for LocalAdapter.php "code review"

Pull Request for Issue # .

This is new to me.
If I did it wrong please let me know.

Summary of Changes
I changed fixed translation to translatable strings

Testing Instructions

"code review"
It's only changed in the code.
I converted the fixed value of the error message into a translatable string

Actual result BEFORE applying this Pull Request
In localAdapter.php there was untranslatable text.

Expected result AFTER applying this Pull Request
In localAdapter.php the text is now translatable using string

### Link to documentations
Please select:
- [X ] No documentation changes for docs.joomla.org needed

- [ X] No documentation changes for manual.joomla.org needed
